src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentTypeFinder_Remote.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
>// [](start = 0, length = 2)

#nullable enable

Reply:
will add in https://github.com/dotnet/roslyn/pull/43804